### PR TITLE
chore: update tab bar icons for selected unselected states

### PR DESCRIPTION
**Pull request by saeedbashir**
Update the tan bar icons for selected and unselected states as per the design team feedback 
image

### CODE REVIEW

**Comment by rnr** [comment]
@saeedbashir 
If we want to be mergeable with upstream, we don't need to replace the icons in that PR, right? We could split the tab icons into _`acive` and `_inactive`, but leave the upstream icons there. And replace them with a Whitelabvel script. What do you think?

**Comment by moiz994** [comment]
@rnr agreed, if we can easily do a white label script that would be good since there is a difference in icons used by schema and 2U. 
@HamzaIsrar12 did we decide on the discovery icon?

**Comment by saeedbashir** [comment]
> @rnr agreed, if we can easily do a white label script that would be good since there is a difference in icons used by schema and 2U. @HamzaIsrar12 did we decide on the discovery icon?

In that case, we will have the same icon for the selected/unselected states for the upstream which doesn't looks good even for making the case. I believe we have already discussed it with Axim and they were ok with the approach. The only thing that will come under discussion while pushing to upstream will be the icons for tabs.

In my opinion. we will update the icons in the codebase, and while pushing it to upstream we will see whether the like the icons or not. If not then we will have different copies for upstream and edX.

**Comment by rnr** [comment]
> > @rnr agreed, if we can easily do a white label script that would be good since there is a difference in icons used by schema and 2U. @HamzaIsrar12 did we decide on the discovery icon?
> 
> In that case, we will have the same icon for the selected/unselected states for the upstream which doesn't looks good even for making the case. I believe we have already discussed it with Axim and they were ok with the approach. The only thing that will come under discussion while pushing to upstream will be the icons for tabs.
> 
> In my opinion. we will update the icons in the codebase, and while pushing it to upstream we will see whether the like the icons or not. If not then we will have different copies for upstream and edX.

@saeedbashir 
if "I believe we have already discussed it with Axim and they were ok with the approach" then there is not a problem to have "the same icon for the selected/unselected states" - when they decided which icons they want they will just replace it. But all this time upstream icon will look as earlier.

**Comment by moiz994** [comment]
@rnr I think we can use the same icon that Schema is using. I also think their icon and our icon is the same as per the figma file. For them, the selected and unselected state is a value added for schema. When this PR goes to RG, they can either accept our icons or we can accept theirs no need to maintain different icon sets. It might add complexity for now. 

We can look at a white label script later.

**Review by rnr — APPROVED** [review]

File: Core/Core/SwiftGen/Assets.swift [hunk 1]
```diff
@@ -76,9 +76,12 @@ public enum CoreAssets {
...
-  public static let discovery = ImageAsset(name: "discovery")
-  public static let learn = ImageAsset(name: "learn")
-  public static let profile = ImageAsset(name: "profile")
+  public static let discoverActive = ImageAsset(name: "discover_active")
+  public static let discoverInactive = ImageAsset(name: "discover_inactive")
+  public static let learnActive = ImageAsset(name: "learn_active")
+  public static let learnInactive = ImageAsset(name: "learn_inactive")
+  public static let profileActive = ImageAsset(name: "profile_active")
+  public static let profileInactive = ImageAsset(name: "profile_inactive")
   public static let programs = ImageAsset(name: "programs")
```
Comment (shafqat-muneer) [inline Core/Core/SwiftGen/Assets.swift:85]: There are two modes of TabView mentioned in `MainScreenView.swift`
- list
- gallery

In list view, we should also add selected and unselected states:
```
.tabItem {
  CoreAssets.programs.swiftUIImage.renderingMode(.template)
  Text(CoreLocalization.Mainscreen.programs)
}
```

If we are not going to maintain list view then we should entirely remove list view related icons and changes.

WDYT?

**Review by shafqat-muneer — CHANGES_REQUESTED** [review]

File: Core/Core/SwiftGen/Assets.swift (hunk 1 above)
Comment (saeedbashir) [reply Core/Core/SwiftGen/Assets.swift:85]: The list view is not our need nor the axim, both providers are using gallery view. If any provider wants to use it, they can configure (open PR) according to their need.

**Review by shafqat-muneer — APPROVED** [review]

**Pull request merged**